src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1278,9 +1278,22 @@ impl Coordinator {
         plan::DropObjectsPlan {
             drop_ids,
             object_type,
-            referenced_ids: _,
+            referenced_ids,
         }: plan::DropObjectsPlan,
     ) -> Result<ExecuteResponse, AdapterError> {
+        let referenced_ids_hashset = referenced_ids.iter().collect::<HashSet<_>>();
+        let mut objects = Vec::new();
+        for obj_id in &drop_ids {
+            if !referenced_ids_hashset.contains(&obj_id) {
+                let object_info = self.get_object_info(obj_id, session)?;
+                objects.push(object_info);
+            }
+        }
+
+        if !objects.is_empty() {
```

Comment:
upon testing -- it makes sense to add this to remove empty notices like: 
`NoticeResponse {"fields":[{"typ":"S","value":"NOTICE"},{"typ":"C","value":"00000"},{"typ":"M","value":"drop cascades to 0 other objects"}]}`